src/main/java/seedu/trackbeau/ui/CustomerCard.java

```diff
@@ -52,9 +60,17 @@ public CustomerCard(Customer customer, int displayedIndex) {
         phone.setText(customer.getPhone().value);
         address.setText(customer.getAddress().value);
         email.setText(customer.getEmail().value);
-        customer.getTags().stream()
+        skinType.setText("Skin Type : " + customer.getSkinType().value);
+        hairType.setText("Hair Type: " + customer.getHairType().value);
```

Comment:
I think there needs to be a consistency in the formatting of the text like having a space before ":" or no spacing?

Reply:
chose no spacing, edited